WARNING: unknown parameter 'print_model' at module 'opt'
Legal parameters are:
  dump_benchmarks (bool) (default: false)
  elim_01 (bool) (default: true)
  enable_sat (bool) (default: true)
  enable_sls (bool) (default: false)
  maxres.add_upper_bound_block (bool) (default: false)
  maxres.hill_climb (bool) (default: true)
  maxres.max_core_size (unsigned int) (default: 3)
  maxres.max_correction_set_size (unsigned int) (default: 3)
  maxres.max_num_cores (unsigned int) (default: 4294967295)
  maxres.maximize_assignment (bool) (default: false)
  maxres.pivot_on_correction_set (bool) (default: true)
  maxres.wmax (bool) (default: false)
  maxsat_engine (symbol) (default: maxres)
  optsmt_engine (symbol) (default: basic)
  pb.compile_equality (bool) (default: false)
  pp.neat (bool) (default: true)
  priority (symbol) (default: lex)
  rlimit (unsigned int) (default: 0)
  solution_prefix (symbol) (default: None)
  timeout (unsigned int) (default: 4294967295)

Exception in thread "main" java.lang.UnsatisfiedLinkError: com.microsoft.z3.Native.INTERNALgetErrorMsgEx(JI)Ljava/lang/String;
	at com.microsoft.z3.Native.INTERNALgetErrorMsgEx(Native Method)
	at com.microsoft.z3.Native.fixedpointQuery(Native.java:3330)
	at com.microsoft.z3.Fixedpoint.query(Fixedpoint.java:126)
	at com.horndroid.z3.FSEngine.executeAllQueries(FSEngine.java:292)
	at com.horndroid.executors.HorndroidExecutor.executeQueries(HorndroidExecutor.java:153)
	at com.horndroid.executors.HorndroidExecutor.processFiles(HorndroidExecutor.java:288)
	at com.horndroid.executors.HorndroidExecutor.execute(HorndroidExecutor.java:304)
	at com.horndroid.Main.main(Main.java:81)
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00007fbb7f7a1000, 12288, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 12288 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /media/data/horndroid_files/hs_err_pid11897.log
